packages/lib-classifier/src/plugins/tasks/TextTask/models/TextTask.js

```diff
@@ -0,0 +1,12 @@
+import { types } from 'mobx-state-tree'
+
+const TextTask = types.model('TextTask', {
+  help: types.optional(types.string, ''),
+  instruction: types.string,
+  required: types.maybe(types.boolean), // Should this be an optional type with the default to true?
```

Comment:
There definitely should be a default here. Why would the default be true and not false? The project builder doesn't default to required being true.

Reply:
I think I copied this comment over from the single choice task model by mistake.